docs/zh-cn/custom_quantum_functions.md

```diff
@@ -286,15 +311,15 @@ void suspend_wakeup_init_user(void) {
 ### 键盘 挂起/唤醒 函数文档
 
 * 键盘/修订: `void suspend_power_down_kb(void)` 和`void suspend_wakeup_init_user(void)`
-* 布局: `void suspend_power_down_kb(void)` 和 `void suspend_wakeup_init_user(void)`
+* 映射: `void suspend_power_down_kb(void)` 和 `void suspend_wakeup_init_user(void)`
 
 # 层改变代码
 
 每当层改变这个就运行代码。这对于层指示或自定义层处理很有用。
 
 ### `layer_state_set_*` 示例实现
 
-本例使用了Planck键盘示范了如何设置 [RGB背光灯](feature_rgblight.md)使之与层对应
+本例使用了Planck键盘示范了如何设置 [RGB背光灯](zh-cn/feature_rgblight.md)使之与层对应
 
 ```c
 uint32_t layer_state_set_user(uint32_t state) {
```

Comment:
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
```

Also, the  `switch (biton32(state))` should be changed to `switch (get_highest_layer(state))`

Reply:
Thanks!